lib/ansible/modules/system/known_hosts.py

```diff
@@ -145,7 +148,7 @@ def enforce_state(module, params):
                 module.fail_json(msg="Failed to read %s: %s" % \
                                      (path,str(e)))
         try:
-            outf=tempfile.NamedTemporaryFile(dir=os.path.dirname(path))
+            outf = tempfile.NamedTemporaryFile(mode='w+', dir=os.path.dirname(path))
```

Comment:
I normally rebase and put all messages in main commit (which shows in summary by default), but when you are doing several iterations its OK as is. When merging, we normally squash them if you hadn't already.

FYI, https://www.openssh.com/report.html, they use bugzilla.